thoth/common/workflows.py

```diff
@@ -885,3 +885,25 @@ def submit_graph_sync(
         )
 
         return workflow_id
+
+    def submit_thoth_repo_init(
+        self,
+        template_parameters: Optional[Dict[str, str]] = None,
+        workflow_parameters: Optional[Dict[str, Any]] = None,
+    ) -> Optional[str]:
+        """Submit repo initialization workflow."""
+        if not self.openshift.infra_namespace:
+            raise ConfigurationError("Infra namespace was not provided.")
+
+        if not self.openshift.middletier_namespace:
```

Comment:
As this is serving to Users, should this execute in the backend namespace?

Reply:
+1